spec/controllers/linked_data_terms_controller_spec.rb

```diff
@@ -2,6 +2,7 @@
 require 'json'
 
 describe Qa::LinkedDataTermsController, type: :controller do
+  let(:graph_load_failure) { /^\*\*\*\*\*\*\*\* RDF\:\:Graph\#load failure/ }
```

Comment:
Pattern for catching new graph load failure logging.  Exact failure is will be driven by the exception thrown and does not require validation here.  The required validation is that the logging occurred.  This pattern allows for that validation.